CONTRIBUTING.md

```diff
@@ -100,11 +100,12 @@ can submit an issue report [via the web](https://github.com/maxmind/App-GHPT/iss
 .
 This is a good place to send your questions about the usage of this distribution.
 
-## Travis
+## GitHub Actions
```

Comment:
This file is generated by the dzil plugin. Updating it manually won't achieve anything. I've updated my own bundle a bit to handle different CI systems, so it might be something you could crib from.